core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -90,7 +90,7 @@ class BillingRepository(dataSource: SlickDataSource) {
         .landingZoneId
     }
 
-  def updateLandingZoneId(projectName: RawlsBillingProjectName, landingZoneId: UUID): Future[Int] =
+  def updateLandingZoneId(projectName: RawlsBillingProjectName, landingZoneId: Option[UUID]): Future[Int] =
```

Comment:
This had to be changed so that `None` could be passed as a value to un-set the landing zone id.